src/js/calculate.js

```diff
@@ -90,46 +91,53 @@ export default {
     },
 
     /**
-     * Runs <setvalue> actions.
-     *
      * @param {CustomEvent} [event] - the event type that triggered the setvalue action.
+     * @param {'setvalue' | 'setgeopoint'} action - the action which should be performed
      */
-    setValue( event ) {
-
-        if ( !event ) {
-            return;
-        }
-
-        if ( !this.form ) {
-            throw new Error( 'Setvalue module not correctly instantiated with form property.' );
-        }
-
-        let nodes = [];
-
+    _getNodesForAction( event, action ) {
         if ( event.type === new events.InstanceFirstLoad().type ) {
             // We ignore relevance for the data-instance-first-load, as that will likely never be what users want for a default value.
             // Do not use getRelatedNodes here, because the obtaining (and caching) of nodes inside repeats is (and should be) disabled at the
             // time this event fires.
             //
-            // We change the order by first evaluating the non-formcontrol setvalue directives (in document order), and then
+            // We change the order by first evaluating the non-formcontrol setvalue/setgeopoint directives (in document order), and then
```

Comment:
"setvalue/setgeopoint directives" -> actions? Similar to what we discussed on at https://github.com/enketo/enketo-transformer/pull/101#discussion_r627570573